tests/src/application/middlewares/auth.spec.ts

```diff
@@ -35,111 +35,106 @@ const makeSut = (): SutTypes => {
 }
 
 describe('Auth Middleware', () => {
-  describe('test the authorization field', () => {
-    it('Should return 401 if no authorization is provided', async () => {
-      const { sut } = makeSut()
+  const httpRequest = makeFakeAuthorizationRequest()
```

Comment:
Nice, good idea to concentrate the positive case of constant for all tests here.